reference/apm/cloud/apm-settings.md

```diff
@@ -51,6 +51,10 @@ If a setting is not supported by {{ech}}, you will get an error message when you
 Some settings that could break your cluster if set incorrectly are blocklisted. The following settings are generally safe in cloud environments. For detailed information about APM settings, check the [APM documentation](/solutions/observability/apm/configure-apm-server.md).
 ::::
 
+### Version 8.19+ and 9.1+ [ec_version_8_19_and_9_1]
+
+`apm-server.sampling.tail.ttl`
+:   Time-to-live (TTL) for trace events stored in the local storage of the APM Server during tail-based sampling. This TTL determines how long trace events are retained in the local storage while waiting for a sampling decision to be made. A greater TTL value increases storage space requirements.
 
 ### Version 8.0+ [ec_version_8_0_3]
```

Comment:
[to reviewer] I'm starting to think we should remove all 7.x and 8.x sections, as this is 9.0 docs. It still makes sense to document 9.0 vs 9.1+ though.

Reply:
The entire (confusing) section may be removed as part of https://github.com/elastic/apm-server/issues/13602